### PR TITLE
Audits Tactical and Army Jackets

### PR DESCRIPTION
**Pull request by DoctorBoomstick**
#### Summary
None
#### Purpose of change
While playing a maul run, seeing the sights, getting beaten by zombies like an uncooperative vending machine that wouldn’t dispense the Twinkie they’d ordered, and generally running for my life, I stumbled into the sporting goods store. There, to my delight, I made a discovery: we had tactical jackets! I was enthused. I was excited. I was pumped to pick up the nearest one, hold it in my hands (while being mauled to death), turn it over, and get ready to read all the variant descriptions I found before me.
> Possessing reinforced knees and a hardy construction, this tactical jacket is resplendent in a camouflage pattern of mixed light greens, browns, and beige tones.  With a set of deep pockets and an insulated design, it's an ideal piece for outdoor ventures.

I will present one brownie to whoever can spot the error in the above description. Outside of the obvious, all the variants for the tactical jackets, when I went file diving, were, essentially, just copies of the descriptions for the camouflaged pants variants and tactical pants. Allow me to demonstrate.

- Alpine-Camo Tactical Jacket
> The companion of aspiring mountaineers and alpine hunters, these brown, gray, and white dyed tactical jacket possess a network of jagged shapes and mottled lines, designed to break up the wearer's profile and better conceal them against the background landscape of mountains.
- Alpine-Camo pants 
> The companion of aspiring mountaineers and alpine hunters, these brown, gray, and white dyed pants possess a network of jagged shapes and mottled lines, designed to break up the wearer's profile and better conceal them against the background landscape of mountains.
- Black-Camo Tactical Jacket
> Dyed a deep hue of urban black, this tactical jacket is designed to assist a wearer in concealing themselves within nighttime environments and low-light conditions.  While it's a simple tactical jacket given a stealth-black paint job and don't hold a candle to their more marshal brothers, they still look cool and are great at making you feel like you're Spetsnaz.
- Black-Camo Pants
> Dyed a deep hue of urban black, this pair of hardwearing pants is designed to assist a wearer in concealing themselves within nighttime environments and low-light conditions.  While they're simple civilian trousers given a stealth-black paint job and don't hold a candle to their more marshal brothers, they still look cool and are great at making you feel like you're Spetsnaz.

So on and so forth. While copying descriptions from one item to another isn’t inherently bad, grammar issues are, and I felt like we could cook up a few better descriptions for these items, as it’s not particularly interesting to see the same descriptions and names transplanted from one item to another.
In the process of redoing the descriptions, I also gave a touch-up to the tactical jacket as a whole. This is a tactical jacket. We need an overwhelming amount of pockets; otherwise, where are you going to stash your all-important operational snickers bar, tactical Teddy, and mission-appropriate potted plant?
As I was working on the above, however, I looked towards the army jacket and saw that it was quite, by this point, antiquated in its implementation, and set to updating it as well with the aid of my own M65 field jacket.
#### Describe the solution
I have:
- Updated and rewrote the descriptions for the tactical jacket while also auditing its stats.
- Updated the properties of the army jacket and added cosmetic variants to it based on attainable M65 jackets I found through browsing.
#### Describe alternatives you've considered
I considered renaming the base army jacket to the M65 strait away, but considered that giving the proper designations to clothing items is one bridge we really, really don’t want to cross.
In a similar vein, I considered adding in jacket variants that weren’t strictly M65 coats, such as Pattern 68/85 combat smocks and the like, but figured that other militaria would be too rare to pop up. Plus, I thought that forcing my grimy surplus addiction on the wider CDDA world was probably ill advised.
#### Testing
To be done later, when it’s not 1:34 am.
#### Additional context
For sake of transparency, I modelled the army jacket’s pocket statistics off of measuring my own M65’s pockets; however, as may be expected, it’s sort of hard to gauge precise statistics when you’re blind and relying on an AI image describer to read the tape measure for you. I’m relatively confident that this shouldn’t be an issue, but I figured I’d put it down just in case there were any glaring auditees.

### CODE REVIEW

**Comment by Hyperseeker** [comment]
I was under the impression that the army jacket was meant to be a modern, in-service (at least at one point) U.S. Army coat, such as [this one](https://enemymilitaria.com/product/us-army-dcu-jacket-double-patched-to-1lt-essig-cavlary-officer-3rd-inf-div-1st-armor-div/). The amount of pockets checks out; M65 appears from cursory overview to be two pockets short.

For what it's worth, I'd modelled the replacement in Armory after the [ACU coat](https://www.propper.com/military-uniforms/army-uniform/acu-uniform/standard-acu/acu-coat-new-spec.html): the current issue of U.S. Army uniform. The coat *technically* has six pockets, but two of them are for pens and such, for when you need to mark something on your map while on a mission. (The ACU pants, by the way, have eight pockets in total, two of which are ankle pockets which can fit a single 30-round M4 magazine each.)

The reason I went with the ACU is, well, because it's current-issue. In-game, the Cataclysm happened... literally yesterday, if I recall the latest lore correctly? So, whatever gear the Army has now, in the real world, is what the *DDA* Army would have when the sheet hit the fan.

Also, for what it's worth, at least the latest (ACU) Army uniform has slots for padding for elbows and knees.

**Comment by Hyperseeker** [comment]
Actually, don't mind me. I forgot that I was replacing not `jacket_army` but `jacket_army_modern`, the latter being the combat blouse. It would make sense that it would be limited to (mostly) military spawns: even with surplus around (and its own manufacturer selling it), it's still primarily an item of uniform.

M65 looks to be more of a fashion item today, what with the reproduction, so it would make sense for it to spawn with civilians. For what it's worth, in that case, I think it wouldn't be suitable for military spawns, since it's very much not a uniform for anyone anymore. (Maybe add it to the Old Veteran profession?)

File: data/json/items/armor/coats.json [hunk 1]
```diff
@@ -890,68 +890,185 @@
...
+        "description": "Tried and true, the US military's antiquated 4-tone woodland camouflage pattern covers this particular jacket: large, high-contrast patches of green, black, brown, and sand hues, all rather faded from years and ware.",
+        "color": "green",
+        "weight": 25,
+        "append": true
+      },
+      {
+        "id": "m65_desert",
+        "//": "Intended to be US DCU camo.",
+        "name": { "str": "desert-camo combat jacket" },
+        "description": "This jacket's brass zipper runs from collar to hem, a river of metallic brown nitting together the coat's arid-hued lapels, printed with a 3-tone mix of pale green, dark brown, and beige.  Looking at it, you wonder how many zombies are wandering about the gulf right now.",
```
Comment (akrieger) [inline data/json/items/armor/coats.json:998]:
```suggestion
        "description": "This jacket's brass zipper runs from collar to hem, a river of metallic brown knitting together the coat's arid-hued lapels, printed with a 3-tone mix of pale green, dark brown, and beige.  Looking at it, you wonder how many zombies are wandering about the gulf right now.",
```

File: data/json/items/armor/coats.json [hunk 2]
```diff
@@ -3505,163 +3622,217 @@
...
         "color": "green",
-        "weight": 6
+        "weight": 30,
+        "append": true
       },
       {
         "id": "tacjacket_jungle",
         "name": { "str": "jungle-camo tactical jacket" },
-        "description": "Similar in color to the old frog skin uniforms issued to Marine Raiders during the Second World War, this warm, comfortable tactical jacket is dyed a greenish gray hue.  Completely conventional in every other respect, the patterning would be incredibly helpful if you were planning to take a trip down to the Amazon any time soon.",
+        "description": "Bright and dark green shades meet within the patterning of this tactical jacket, embellished by tones of earthen brown and other natural hues.  The trees of the Amazon will speak your native tongue as you stalk around, waging gorilla warfare against the undead, with this jacket on.  It's just a shame that the rainforest is quite a walk away.",
```
Comment (akrieger) [inline data/json/items/armor/coats.json:3736]:
```suggestion
        "description": "Bright and dark green shades meet within the patterning of this tactical jacket, embellished by tones of earthen brown and other natural hues.  The trees of the Amazon will speak your native tongue as you stalk around, waging guerilla warfare against the undead, with this jacket on.  It's just a shame that the rainforest is quite a walk away.",
```

File: data/json/items/armor/coats.json [hunk 3]
```diff
@@ -3505,163 +3622,217 @@
...
         "color": "green",
-        "weight": 6
+        "weight": 15,
+        "append": true
       },
       {
         "id": "tacjacket_marshland",
         "name": { "str": "marshland-camo tactical jacket" },
-        "description": "The faithful friend of waterfowl hunters, this tactical jacket is printed with a brown, gray, tan, and green camouflage pattern designed to blend in with the vegetation and reeds present in swamps and marshlands.",
+        "description": "A staple companion of gamesmen who's trade lies within the watery recesses of wetlands, this water-resistant jacket is festooned with profile-disrupting lines and patterns printed in tan, grey, green, and black hues.",
```
Comment (akrieger) [inline data/json/items/armor/coats.json:3752]:
```suggestion
        "description": "A staple companion of gamesmen whose trade lies within the watery recesses of wetlands, this water-resistant jacket is festooned with profile-disrupting lines and patterns printed in tan, grey, green, and black hues.",
```

File: data/json/items/armor/coats.json [hunk 4]
```diff
@@ -3505,163 +3622,217 @@
...
+        "weight": 5,
+        "append": true
       },
       {
         "id": "tacjacket_grey",
         "name": { "str": "wolf tactical jacket" },
-        "description": "This particular tactical jacket is dyed a muted, beige-like shade of wolf gray.",
-        "color": "blue",
-        "weight": 6
+        "description": "Suttle and unassuming, save for the purely Ludacris number of pockets it's coated in, this hardy overgarment's outer shell is dyed a muted tone of grey, verging into beige.",
```
Comment (akrieger) [inline data/json/items/armor/coats.json:3808]:
```suggestion
        "description": "Subtle and unassuming, save for the purely ludicrous number of pockets it's coated in, this hardy overgarment's outer shell is dyed a muted tone of grey verging into beige.",
```

File: data/json/items/armor/coats.json [hunk 5]
```diff
@@ -3505,163 +3622,217 @@
...
+        "weight": 10,
+        "append": true
       },
       {
         "id": "tacjacket_river",
         "name": { "str": "river-camo tactical jacket" },
-        "description": "Employed by anglers and river-side hunters for camouflage in freshwater habitats, this tactical jacket incorporates shades of green, brown, blue, and gray, along with sizable pockets and a construction favoring heat retention.",
-        "color": "green",
-        "weight": 6
+        "description": "The fabric of this particular jacket is festooned with a camouflage pattern — a muddle of greens, light blues, browns, and grey — that would make it at home with any river-based huntsman or angler, with its water-repellent construction insuring that one's possessions are protected from river spray.",
```
Comment (akrieger) [inline data/json/items/armor/coats.json:3792]:
```suggestion
        "description": "The fabric of this particular jacket is festooned with a camouflage pattern — a muddle of greens, light blues, browns, and grey — that would make it at home with any river-based huntsman or angler. Its water-repellent construction ensures that one's possessions are protected from river spray.",
```

File: data/json/items/armor/coats.json [hunk 6]
```diff
@@ -890,68 +890,185 @@
...
+        "description": "Dyed in a solid shade of rugged, earthen brown with a brass main zip and press-studs, this is a relatively hansom coat, so far as reproduction military equipment goes.  Obviously, the would-be soldier, prepper, and general wearer would most appreciate its utility in going undercover among the insurgent coyote population.",
+        "color": "brown",
+        "weight": 20,
+        "append": true
+      },
+      {
+        "id": "m65_player",
+        "//": "Only used as a cosmetic variant for player professions that start with army jackets: currently the old veteran.",
+        "name": { "str": "woodland-camo combat jacket" },
+        "description": "Dyed in a 4-tone blend of worn green, black, sand, and brown hues, this particular overcoat had been your own issued service jacket when you''d still been in the forces, with your name stitched upon the right lapel and insignia pinned to the appropriate Sholder board.",
```
Comment (akrieger) [inline data/json/items/armor/coats.json:1066]:
```suggestion
        "description": "Dyed in a 4-tone blend of worn green, black, sand, and brown hues, this particular overcoat had been your own issued service jacket when you''d still been in the forces, with your name stitched upon the right lapel and insignia pinned to the appropriate shoulder board.",
```

**Review by akrieger — COMMENTED** [review]
Spelling fixes and broke one sentence up for being too run on to my tongue.

**Pull request merged**